functions.py

```diff
@@ -2,159 +2,159 @@
+                im.save('resized_images/'+name)
+                print "- " + name + " : " + "DONE"
+                raw_input("Saved in resized_images.")
+            except:
+                raw_input('Error..')
 
-def cut_image():
-    print '''
-    =====================
-    CUT IMAGE
-    =====================
-    1. File
-    2. Folder
-    3. Close
-    =====================
-    '''
-    choice = raw_input(">> ")
-    if choice == "1":
-        directory = raw_input('Directory to file: ')
-        width = input('Width: ')
-        height = input('Height: ')
-        name = raw_input('Save as: ')
-        im = Image.open(directory)
-        im = im.resize((width, height))
-        if not os.path.exists('resized_images'):
-            os.mkdir('resized_images')
-        try:
+        elif choice == "2":
+            f_directory = raw_input('Directory to folder: ')
+            width = input('Width: ')
+            height = input('Height: ')
+            try:
+                fieldlist = os.listdir(f_directory)
+            except OSError:
+                raw_input('Folder does not exist.')
+                sys.exit()
+            extensions = ['JPG', 'PNG', 'JPEG', 'GIF']
+            if not os.path.exists('resized_images'):
+                os.mkdir('resized_images')
             print "Size: " + str(width) + " x " + str(height)
-            im.save('resized_images/'+name)
-            print "- " + name + " : " + "DONE"
+            for item in fieldlist:
+                directory = f_directory+ "\\" + item
+                im = Image.open(directory)
+                if im.format in extensions:
+                    im = im.resize((width, height))
+                    im.save('resized_images/'+item)
+                    print "- " + item + " : " + "DONE"
             raw_input("Saved in resized_images.")
-        except:
-            raw_input('Error..')
-
-    elif choice == "2":
-        f_directory = raw_input('Directory to folder: ')
-        width = input('Width: ')
-        height = input('Height: ')
-        try:
-            fieldlist = os.listdir(f_directory)
-        except OSError:
-            raw_input('Folder does not exist.')
+        elif choice == "3":
+            os.system('clear')
             sys.exit()
-        extensions = ['JPG', 'PNG', 'JPEG', 'GIF']
-        if not os.path.exists('resized_images'):
-            os.mkdir('resized_images')
-        print "Size: " + str(width) + " x " + str(height)
-        for item in fieldlist:
-            directory = f_directory+ "\\" + item
+            
+    def change_format(self):
+        print '''
+        =====================
+        Change Format
+        =====================
+        1. File
+        2. Folder
+        3. Close
+        =====================
+        '''
+        choice = raw_input(">> ")
+        if choice == "1":
+            directory = raw_input('Directory to file: ')
+            ext = raw_input('Save as: ')
```

Comment:
Input handling should be extracted to `Main` class. Here we should just do operations on the images not read the input.